spec/shadow/index.html

```diff
@@ -1137,6 +1137,20 @@
       </section>
 
       <section>
+        <h3>Focus</h3>
+      <p>A <a>shadow host</a> can delegate focus to its <a>shadow root</a> by assigning a boolean <a href="#widl-ShadowRootInit-delegatesFocus">delegatesFocus</a> flag to be true in <a href="#idl-def-ShadowRootInit">ShadowRootInit</a> dictionary. If omitted, a <a>shadow host</a> does not delegate focus to its shadow root, and the shadow host itself can be focusable.</p>
+        <p>When a shadow host <var>HOST</var> delegates focus, user agent <strong>must</strong> behave as follows.
+          <ol>
+            <li>In <a>sequential focus navigation</a>, <var>HOST</var> itself will be skipped. See the next secition for the formal definition.</li>
+            <li>When <var>HOST</var> is focused by <code>focus()</code> method or <code>autofocus</code> attribute: The first <a>focusable area</a> in the tree order in <var>HOST</var>'s <a>shadow tree</a> gets focus.</li>
```

Comment:
no, the first one in the focus navigation order of its shadow root.
Replaced the sentence.